agent/api/task/task_test.go

```diff
@@ -399,13 +388,6 @@ func TestDockerHostConfigPortBinding(t *testing.T) {
 					if !reflect.DeepEqual(tc.testTask.Containers[0].ContainerPortRangeMap, tc.expectedContainerPortRangeMap) {
 						t.Error("Expected container port range map to be resolved, was: ", tc.testTask.Containers[0].GetContainerPortRangeMap())
 					}
-				} else {
-					// Verify ECS Agent assigned host port range are within the dynamic host port range
```

Comment:
__NEW!__: Remove host port range validation in `TestDockerHostConfigSCBridgeMode` as ports should be validated before returning back from  `utils.GetHostPortRange()`